tests/main/views/test_marketplace.py

```diff
@@ -1671,13 +1671,13 @@ def test_should_render_suggestions_for_0_results(self):
 
         res = self.client.get('/digital-outcomes-and-specialists/opportunities')
         assert res.status_code == 200
-        suggestion = re.findall(r'<p>Suggestions:<\/p>', res.get_data(as_text=True))
+        suggestion = re.findall(r'<div class="search-suggestion">', res.get_data(as_text=True))
```

Comment:
This feels a bit fragile. If we add a new class to this div then the test will break.

Can we use xpath `contains(@class, 'search-suggestion')` maybe instead?

Reply:
Good point, will fix